src/helm/benchmark/presentation/run_entries_lite_20240424.conf

```diff
@@ -0,0 +1,133 @@
+# HELM scenarios.
+
+entries: [
+  # NarrativeQA
+  {description: "narrative_qa:model=text,follow_the_format_instructions=instruct", priority: 1}
```

Comment:
nit: could we call this `follow_format_instructions` just to make a tad shorter?

Reply:
Changed.